components/automate-cli/cmd/chef-automate/ha_node_add_onprem.go

```diff
@@ -196,7 +202,14 @@ func (ani *AddNodeOnPremImpl) runDeploy() error {
 		return err
 	}
 	argsdeploy := []string{"-y"}
-	return ani.nodeUtils.executeAutomateClusterCtlCommandAsync("deploy", argsdeploy, upgradeHaHelpDoc)
+	err = ani.nodeUtils.executeAutomateClusterCtlCommandAsync("deploy", argsdeploy, upgradeHaHelpDoc)
+	syncErr := ani.nodeUtils.syncConfigToAllNodes()
+	if syncErr != nil {
+		if err != nil {
+			return errors.Wrap(err, syncErr.Error())
+		}
+	}
+	return syncErr
```

Comment:
```suggestion
	if syncErr != nil {
		if err != nil {
			return errors.Wrap(err, syncErr.Error())
		}
		return syncErr
	}
	return err
}
```
Follow this same logic that you have used at other places

Reply:
Updated